src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,131 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.math.BigDecimal;
+import java.util.List;
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@Transactional
+class MenuServiceTest {
+
+    @Autowired
+    private MenuService menuService;
+
+    @Autowired
+    private MenuGroupDao menuGroupDao;
+
+    @Autowired
+    private MenuDao menuDao;
+
+    @Autowired
+    private ProductDao productDao;
+
+    private Long validMenuGroupId;
+    private Long validProductId;
+
+    @BeforeEach
+    void setUp() {
+        final MenuGroup menuGroup = menuGroupDao.save(new MenuGroup("추천메뉴"));
+        validMenuGroupId = menuGroup.getId();
+        final Product product = productDao.save(new Product("후라이드", new BigDecimal(16000)));
+        validProductId = product.getId();
+    }
+
+    @DisplayName("메뉴를 저장한다.")
+    @Test
+    void create() {
+        // given
+        final MenuProduct menuProduct = new MenuProduct(validProductId, 3);
+        final Menu menu = new Menu("후라후라후라", new BigDecimal(19000), validMenuGroupId, List.of(menuProduct));
+
+        // when
+        final Menu savedMenu = menuService.create(menu);
+
+        // then
+        assertAll(
+                () -> assertThat(savedMenu).usingRecursiveComparison()
+                        .ignoringFields("id", "price", "menuProducts")
+                        .isEqualTo(menu),
+                () -> assertThat(savedMenu.getMenuProducts()).hasSize(1)
+        );
```

Comment:
```suggestion
        assertAll(() -> {
                assertThat(savedMenu).usingRecursiveComparison()
                        .ignoringFields("id", "price", "menuProducts")
                        .isEqualTo(menu);
                assertThat(savedMenu.getMenuProducts()).hasSize(1);
        });
```
a: 이런식으로도 할 수 있겠네요 ㅎㅎ

Reply:
혹시 이렇게 하는 것의 장점이 있을까요?? 기존의 방식을 이용하면 두 개의 케이스가 다 돌아가고 파랑의 제안대로하면 순차적으로 진행하다가 기대값이 아니면 멈춘다는 점이 장점일까요? 궁금해요!

Reply:
앗 저는 단순히 스타일 차이를 얘기했었는데 아스피덕에 반대로 차이점을 배워가네요 ㅎㅎㅎ 
그냥 이런 방법도 있다~ 정도로만 생각해주시면 될 것 같아요!